### DISCUSSION TITLE
MIMIC-III validity of ICD9 codes?

### DISCUSSION THREAD

**Discussion by eyamga** [MIT-LCP/mimic-code#1395] [MIMIC-III]
Hi,

I have a question regarding
1) procedures_icd
2) diagnoses_icd

Do these codes reflect events or diagnoses having occurred during the hospital stay?
i.e. if a patient has a procedure recorded during a hospital stay, can we assume that the PCI has occurred during that stay hospital stay?  in other words, is it possible that this procedure icd code reflects something that has occurred during a previous hospital admission? 

i.e.2.  patient has a diagnosis (example diabetes) recorded during a hospital stay, my belief is that we cannot assume that this means that this diagnosis has occurred during that hospital stay. 

Thank you for the clarification

Eric

**Comment by alistairewj** [comment]
The diagnoses and the procedures will certainly be associated with the given hospital stay. This is very clear cut for procedures. For diagnoses, it's a similar story, but there's no guarantee the diagnosis *started* at that stay. Your example of diabetes is a good one - yes the patient may have had diabetes during the hospital stay, but more than likely they also had it before the hospital stay.
